### DISCUSSION TITLE
unable to run the example on my local machine via go run main.go

### DISCUSSION THREAD

**Discussion by rockey5520** [charmbracelet/bubbletea#528] [Q&A] [answered]
Hello,

I am trying an example at this below location and wonder any help if you can help me understand why I couldn't run the program using `go run main.go` but was able to run the binary which is the outcome of go build

https://github.com/charmbracelet/bubbletea/tree/master/examples/list-fancy

![image](https://user-images.githubusercontent.com/1937561/195065043-df5a23cf-27e8-4c2c-94be-e79d096d6eb7.png)

**Comment by muesli** [comment, marked as answer]
`list-fancy` consists of multiple source files (ending in `.go`) and you're trying to run only a single one of them.

Try:

```bash
go run .
```

or

```bash
go run *.go
```
